api/auth.ts

```diff
@@ -40,7 +40,7 @@ export const guestLogin = async (name: string): Promise<User> => {
 };
 
 export const isAuthenticated = () => {
-  return !!auth.getToken() && auth.getToken() != "";
+  return !!auth.getToken();
```

Comment:
@hmeister10 Did we figure out why this was returning true for jatin?

Reply:
Also look at the constructor, undefined token will still get set no?

Reply:
no the constructor also calls the `setToken` method, which checks before setting the cookie